index.html

```diff
@@ -184,5 +184,16 @@ <h3 class="about-header">About<br>Myself</h3>
                 <img src="req/sep.png" alt="#">
             </div>
     </section>
+    <section class="form-container">
+        <img src="req/contact form shapes-mobile.png" class="form-container__img">
+        <h3 class="contact-header">Contact me</h3>
+            <p class="contact-p">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/mayznbvq" method="post" class="form-container__form">
+            <input type="text" name="username" placeholder="Please insert your name" class="box-input" required>
```

Comment:
- Kindly, limit the name field input length to  `maxlength="30"` to satisfy the [requirment](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md).👌👍